myst_nbs/splines/spline.myst.md

```diff
@@ -6,17 +6,17 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: Python 3.9.10 ('pymc-dev-py39')
   language: python
   name: python3
 ---
 
 # Splines in PyMC3
 
-:::{post} Oct 8, 2021 
-:tags: patsy, pymc3.Deterministic, pymc3.Exponential, pymc3.Model, pymc3.Normal, regression, spline
+:::{post} May 6, 2022 
+:tags: patsy, pymc.Deterministic, pymc.Exponential, pymc.Model, pymc.Normal, regression, spline
 :category: beginner
-:author: Joshua Cook, Tyler James Burch
+:author: Joshua Cook updated by Tyler James Burch, Chris Fonnesbeck
```

Comment:
This is metadata used to generate the citation advise at the bottom. It should only have names and exclude those who have only re-executed the notebooks without making changes. The Authors section should always be present and list everyone and the changes they did.

We discussed that both in an issue, in the docs channel in slack and in a couple doc meetings. Like I often say, I do not have strong feelings about what was chosen and would have been perfectly happy (or even more happy) with another choice or with changing this now. What I do have strong feelings about is that all notebooks do the same because otherwise some people don't dare add their name at all whereas others do which only increases already present inequalities. Therefore changing that would mean changing the style guide and updating all the notebooks that are already in the [Done](https://github.com/pymc-devs/pymc-examples/projects/1) column.